### ISSUE TITLE
From_logit in A2C_discrete.py should be False

### ISSUE THREAD

**Issue by gzlong96** [marload/deep-rl-tf2#5]
In the Actor net, It seems that from_logit should be set to False in tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True) as you added a softmax in the last layer:)
